src/main/kotlin/nl/info/zac/documentcreation/BpmnDocumentCreationService.kt

```diff
@@ -0,0 +1,124 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos, 2024 INFO.nl
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package nl.info.zac.documentcreation
+
+import jakarta.enterprise.context.ApplicationScoped
+import jakarta.enterprise.inject.Instance
+import jakarta.inject.Inject
+import jakarta.ws.rs.HttpMethod
+import net.atos.client.zgw.zrc.model.Zaak
+import net.atos.zac.util.MediaTypes
+import nl.info.client.smartdocuments.model.document.OutputFormat
+import nl.info.client.smartdocuments.model.document.Selection
+import nl.info.client.smartdocuments.model.document.SmartDocument
+import nl.info.client.smartdocuments.model.document.Variables
+import nl.info.client.zgw.util.extractUuid
+import nl.info.zac.authentication.LoggedInUser
+import nl.info.zac.documentcreation.converter.DocumentCreationDataConverter
+import nl.info.zac.documentcreation.model.BpmnDocumentCreationDataAttended
+import nl.info.zac.documentcreation.model.DocumentCreationAttendedResponse
+import nl.info.zac.identity.model.getFullName
+import nl.info.zac.smartdocuments.SmartDocumentsService
+import nl.info.zac.smartdocuments.SmartDocumentsTemplatesService
+import nl.info.zac.util.AllOpen
+import nl.info.zac.util.NoArgConstructor
+import java.time.ZonedDateTime
+import java.util.UUID
+import java.util.logging.Level
+import java.util.logging.Logger
+
+@NoArgConstructor
+@ApplicationScoped
+@AllOpen
+@Suppress("LongParameterList")
+class BpmnDocumentCreationService @Inject constructor(
```

Comment:
What makes this service `BPMN` (and the other `CMMN`)? I see no BPMN logic or anything here except for the redirect URL. Seems a bit arbitrary to me to name this a `BPMN` service? Same with the CMMN one.

Reply:
I would then at least explain in a class comment what makes this BPMN (and CMMN).

Reply:
Added comments